src/packet/secret_key.js

```diff
@@ -199,6 +199,7 @@ SecretKey.prototype.encrypt = function (passphrase) {
   arr.push(crypto.cfb.normalEncrypt(symmetric, key, cleartext, iv));
 
   this.encrypted = util.concatUint8Array(arr);
+  this.isDecrypted = false;
 };
```

Comment:
@evilaliv3 You set `isDecrypted` to false, but it looks to me like the private MPIs are still in the key after the encrypt. If this is correct, I would rather call `clearPrivateMPIs` after encrypt which also sets `isDecrypted`.